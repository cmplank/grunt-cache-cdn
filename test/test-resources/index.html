<!DOCTYPE html>
<html>

<head>
  <meta charset="UTF-8">
  <title>title</title>
  <!-- cdn js libs -->
  <!-- cdn css libs -->
</head>

<body>

</body>

</html>