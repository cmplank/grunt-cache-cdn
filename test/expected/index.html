<!DOCTYPE html>
<html>

<head>
  <meta charset="UTF-8">
  <title>title</title>
  <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.0/jquery.min.js' defer></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js' defer></script>
  <link href='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css'>
</head>

<body>

</body>

</html>